data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+			`	Finally, Desyo is a xenoprogramming expert who studies alien software, and seemingly has a lot of experience with Sheragi code. Apparently, the displays of the Sheragi room of the museum all run in this exotic language, unseen by the usual public.`
+			`	After the brief introduction, they each take their equipment and load it on the cargo bay of your ship. Rakei approaches you after everything is in place.`
+			`	"We want to meet the human archaeologist who uncovered the ruins in the Rim. Can you take us there? We will start examining the cube along the way. I think by the time we get there, we'll have something to work with."`
+				accept
+	
+	on complete
+		event "cube decoded" 40 50
+		conversation
+			`It was a long trip, almost halfway across the galaxy. The Hai archaeologists have been friendly to you, giving you updates every time they make a new breakthrough. During this time, the cube, previously thought to be impenetrable, had been disassembled and cleaned. Components that decayed into a mess of oxidized wires and goopy paste over the eons have been identified and removed to substitute for equivalent ones, and a blueprint of the box and its circuits has started to take shape.`
+			`	As soon as you land, you call Albert Foster, who doesn't believe you when you say you got alien archaeologists to help. You then look for the three Hai to figure out how they're going to go unnoticed on a human world. You're surprised to find Eyama wearing long, dark but colorful baggy human clothing with an equally oversized hoodie and a bandanna. You recognize this outfit, very popular with the youth these days, who are generally regarded as delinquents by the previous generation. Additionally, she's carrying two more sets, for her Hai partners as well. It hides her body really well at the cost of making her look like a young boy, and the Hai's relatively short stature doesn't help.`
+			`	"Call me Miss Deus ex Machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`
+			choice
+				`	"That's... clever. It hides you well."`
+				`	"Well, it works, but you look like a child."`
+					goto child
+			`	"Right? I can even hide my tail in the back of the hoodie!" She spins around happily.`
+				goto anyway
+			label child
+			`	"That's not that bad, is it? I always considered myself young at heart anyway!" she giggles a little.`
+			label anyway
+			`	"Anyway, can you take us to Mr. Foster's museum? We'll take the cube and fill him in on the details."`
+			`	You agree, and after they're all ready, you make your way out to the dig site. Surprisingly, the three Hai don't get many weird looks if they walk a bit further behind you. You arrive at the museum and they immediately break character by being very obviously excited to see the exhibits. This probably comes as a relief to onlookers, however, who would normally expect that type of kid to vandalize priceless artifacts.`
```

Comment:
I'd suggest removing `by being very` and replace it with just a comma.
`who would normally expect that type of kid to vandalize priceless artifacts`
The wording's a little strange here, especially `that type of kid`. I can't think of anything better at the moment, however. I would at least consider a change.

Reply:
I'll need a second opinion on this, i also don't know how to improve it.

Reply:
Leaving `by being very` seems fine to me. Perhaps change `that type of kid` to just ` kids`? This part too sounds fine to me as is, though.